Creecros patch 1

creecros opened the pull request.

erwindeclerck commented:
Hi 

I cleaned up the code and changed some things.
1. I replaced the 'title' field by 'multitasktitles' in AutoCreateSubtask.php and 
2. changed the template for the form to show a textarea by using this 'multitasktitles' $param_name
3. changed the screenshot
4. added help-message under the textarea.

- I continued on your master, but perhaps you should make a dev-branch for a while to test it out a bit more thoroughly?
- I'm not sure how to get the success message in the flash-zone. See comment on line 94 of AutoCreateSubtask.php

Seems to work fine for me.

Kind regards,
Erwin

creecros commented:
I'll see if I can figure out the flash bit, but also need to figure out the leaving blank bit, as it doesn't seem to work. I'll merge once I figure it out. Should also add your self as Co author since you put work into it.

creecros commented:
I think the flash message is just simply not stacking and being overwritten by TaskCreationController.php
Fixed the leaving blank option, updated master and updated install files.
